.github/workflows/actions.yaml

```diff
@@ -20,9 +20,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout the code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
       - name: Run gosec to check for security vulnerabilities
-        uses: dell/common-github-actions/gosec-runner@main
+        uses: securego/gosec@master
```

Comment:
Are you ensuring this change is being made across all CSM repositories?

Reply:
Yes this change is to be in sync with karavi-resiliency